src/main/java/core/db/StorageDao.java

```diff
@@ -0,0 +1,15 @@
+package core.db;
+
+public class StorageDao extends Storage {
```

Comment:
```suggestion
public class StorageDao {
```
В данном случае под Storage мы представляем себе базу данных, которая не является java-объектом